spring-kafka/src/main/java/org/springframework/kafka/support/DefaultProducerListener.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2015-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.kafka.support;
+
+import org.apache.kafka.clients.producer.RecordMetadata;
+
+/**
+ * No-op implementation of @link ProducerListener}, to be used as base class for other implementations.
+ *
+ * @author Marius Bogoevici
+ * @author Gary Russell
+ */
+public class DefaultProducerListener<K, V> implements ProducerListener<K, V> {
```

Comment:
Typically in Spring similar classes are called like `...Adapter`. E.g. `ChannelInterceptorAdapter`